_posts/2025-01-15-moving-data-between-JBOD-disks-using-cruise-control.md

```diff
@@ -0,0 +1,546 @@
+---
+layout: post
+title: "Moving data between JBOD disks using Cruise Control"
+date: 2025-01-15
+author: shubham_rawat
+---
+
+Apache Kafka is a platform which provides durability and fault tolerance by storing messages on disks and JBOD storage is one of the storage configuration types supported by Strimzi.
+The JBOD data storage configuration allows Kafka brokers to make use of multiple disks.
+Using JBOD storage, you can increase the data storage capacity for Kafka nodes, which can further lead to performance improvements.
+If you plan to remove a disk that contains partition replicas, the data must be safely moved to other disks first.
+Failing to do so could result in data loss.
+Moving data between the JBOD disks can be done using the `kafka-reassign-partitions.sh` tool which is not very user-friendly, therefore in Strimzi 0.45.0 we introduced the ability to move data between the JBOD disks using Cruise Control.
+
+### Cruise Control to move data between JBOD disks
```

Comment:
Following on the comment above - I don't think we want to go into details for adding volumes and focus on removing them for the rest of the blog post. So I would adjust the title accordingly.